manifests/init.pp

```diff
@@ -113,6 +108,12 @@
     host_aliases => $my_localhost_aliases,
     ip           => $localhost_ip,
   }
+  
+  host { "${::fqdn}" :
```

Comment:
This seems totally unrelated to the target, could you open a new PR or issue explaining this?